catalog/ui/search-ui/simple/src/main/webapp/lib/handlebars-1.0.0/handlebars.js

```diff
@@ -195,8 +194,7 @@ performAction: function anonymous(yytext,yyleng,yylineno,yy,yystate,$$,_$) {
 
 var $0 = $$.length - 1;
 switch (yystate) {
-case 1: return $$[$0-1]; 
-break;
+case 1: return $$[$0-1];
```

Comment:
@vinamartin I don't think so, because when `case 1` executes it will return a value, so it won't look at the code below it